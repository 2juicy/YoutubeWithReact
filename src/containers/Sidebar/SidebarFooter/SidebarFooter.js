import React from "react";
import "./SidebarFooter.scss";

export default function SidebarFooter() {
  return (
    <>
      <div className="footer-block">
        <div>About Press Copyright</div>
        <div>Contact us Creators</div>
        <div>Advertise Developers</div>
      </div>
      <div className="footer-block">
        <div>Terms Privacy</div>
        <div>Policy & Safety</div>
        <div>Test new features</div>
      </div>
      <div className="footer-block copyright">
        <div>Â© 2019 YourTube, LLC</div>
      </div>
    </>
  );
}
