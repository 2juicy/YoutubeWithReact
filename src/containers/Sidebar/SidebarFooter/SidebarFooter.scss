.footer-block {
  padding: 0 0 10px 16px;
  color: #918888;
}

.copyright {
  font-weight: bold;
}
