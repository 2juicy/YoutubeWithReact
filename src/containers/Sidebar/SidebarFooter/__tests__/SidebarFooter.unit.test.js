import React from "react";
import { shallow } from "enzyme";
import SidebarFooter from "../SidebarFooter";

describe("SidebarFooter", () => {
  test("renders", () => {
    const wrapper = shallow(<SidebarFooter />);
    expect(wrapper).toMatchSnapshot();
  });
});
