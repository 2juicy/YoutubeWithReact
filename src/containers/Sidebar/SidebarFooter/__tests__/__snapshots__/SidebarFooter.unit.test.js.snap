// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarFooter renders 1`] = `
<Fragment>
  <div
    className="footer-block"
  >
    <div>
      About Press Copyright
    </div>
    <div>
      Contact us Creators
    </div>
    <div>
      Advertise Developers
    </div>
  </div>
  <div
    className="footer-block"
  >
    <div>
      Terms Privacy
    </div>
    <div>
      Policy & Safety
    </div>
    <div>
      Test new features
    </div>
  </div>
  <div
    className="footer-block copyright"
  >
    <div>
      Â© 2019 YourTube, LLC
    </div>
  </div>
</Fragment>
`;
